src/modules/video_player/index.js

```diff
@@ -57,7 +57,31 @@ function handlePlayerClick() {
 }
 
 function handlePlayerScroll(event) {
+  maybeSeek(event);
+  maybeControlVolume(event);
+}
+
+function maybeSeek (event) {
+  if (!settings.get(SettingIds.SCROLL_VIDEO_SEEK)) return;
+  // Non-Alt scrolling controls volume
+  if (!event.altKey) return;
+
+  // Default seek time is 2 seconds for VODs
+  const delta = event.originalEvent.deltaY > 0 ? -2 : 2;
+
+  const currentPlayer = twitch.getCurrentPlayer();
+  if (!currentPlayer) return;
+
+  currentPlayer.seekTo(currentPlayer.getPosition() + delta);
```

Comment:
how does this code function for live content?

Reply:
I've removed the possibility to do this in live content. It somewhat worked but was very unintuitive.